src/ffi.rs

```diff
@@ -53,7 +53,7 @@ impl<'a> FormatterToken<'a> {
             let mut val = mem::uninitialized();
             check!(ZydisFormatterTokenGetValue(self, &mut ty, &mut val))?;
 
-            let val = CStr::from_ptr(val as *const i8)
+            let val = CStr::from_ptr(val as _)
```

Comment:
I think `*const _` is nicer, you've also done so below. I'm just going to merge your PR and change it myself.